crm_claim/report/crm_claim_report.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015-2017 Odoo S.A.
+# Copyright 2017 Vicent Cubells <vicent.cubells@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models, tools
+
+AVAILABLE_PRIORITIES = [
```

Comment:
Put directly this on the field. I haven't seen this used in any other place. Am I right?